src/pybind/mgr/dashboard/tests/test_erasure_code_profile.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from .. import mgr
+from .helper import ControllerTestCase
+from ..controllers.erasure_code_profile import ErasureCodeProfile
+
+
+class ErasureCodeProfileTest(ControllerTestCase):
+    @classmethod
+    def setup_server(cls):
+        mgr.get.side_effect = lambda key: {
+            'osd_map': mock_osd_map,
```

Comment:
```suggestion
            'osd_map': {
                'erasure_code_profiles': {
                    'test': {
                        'k': '2',
                        'm': '1'
                    }
                }
            },
```

Reply:
> > Not sure why the make check on arm64 failed, the logs doesn't show anything specific to the patch.
> 
> ```
>   File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/tests/test_erasure_code_profile.py", line 12, in <lambda>
>     'osd_map': mock_osd_map,
> NameError: global name 'mock_osd_map' is not defined
> ```
> 
> Check your code again. You can also run the Python unit tests locally via `run-tox.sh` before you push PR's or changes.

Thats a bad patch, apologies for the oversight. Have updated the changes.